docet-core/pom.xml

```diff
@@ -82,17 +82,25 @@
             <artifactId>jackson-mapper-asl</artifactId>
             <version>${libs.jackson.mapper}</version>
         </dependency>
-        <dependency>
-            <groupId>com.itextpdf</groupId>
-            <artifactId>itextpdf</artifactId>
-            <version>${libs.itext}</version>
-            <optional>true</optional>
-        </dependency>
-        <dependency>
-            <groupId>com.itextpdf.tool</groupId>
-            <artifactId>xmlworker</artifactId>
-            <version>${libs.itext}</version>
-            <optional>true</optional>
-        </dependency>
+
+<!--
```

Comment:
maybe it is better to drop these lines or have the dependency explictly declared
I prefer option 2 so it is explicit

Reply:
hum the comment is an unlucky leftover from tests with other pdf libraries... removing it